test/local_integration_test.py

```diff
@@ -125,6 +137,26 @@ def check_bdbag(self, response: bytes):
                     self.assertTrue(num_rows > 1)
                     logger.info(f'BDBag file {file_name} contains {num_rows} rows.')
 
+    def download_file_from_drs_response(self, response_content):
```

Comment:
Please make the signature consistent with the existing methods that consume a response.

Just in general, I find it annoying when contributors partially inject their preferences into the code base. Either change the signature for all methods of the same kind—and be ready to defend your decision—or stick with what's already there. You're not making it easier for anyone by having things your way only in the parts you change. Above all, we value consistency with precedent.

Reply:
I didn't intent to inject anything and I, too, value consistency and will try to implement it going forward. I changed it in all three _def_s that consume the response content to `(..., response_content: bytes, ...)`. That makes it clear that the argument is only the response content (similar to variable name `response_json`, used in some other _def_ in that module).

Reply:
You forgot the type annotation. And please use the name `response`, not `response_content`.